tests/phpunit/elementor/core/base/test-document.php

```diff
@@ -90,7 +90,7 @@ public function test_on_import_replace_dynamic_content() {
 			'post_id' => $post->ID,
 		] );
 
-		$map_old_new_post_ids = [ '35' => '36', '37' => '38' ];
+		$new_ids_map = [ '35' => '36', '37' => '38' ];
```

Comment:
$new_ids_map was associative array [ post_ids => ..., term_ids => ..., ] at the beginning, why did you change it and still used the same name?
I probably missed the part where you filter the array.

Reply:
Changed back.